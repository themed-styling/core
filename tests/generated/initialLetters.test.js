import 'jest-styled-components'

import { initialLetters } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(initialLetters, 'initialLetters')
