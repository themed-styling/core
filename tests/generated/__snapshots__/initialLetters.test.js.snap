// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initialLetters renders 0 1`] = `
Array [
  .c0 {
  initial-letters: 0;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: 0;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: 0;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`initialLetters renders function 1`] = `
Array [
  .c0 {
  initial-letters: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: has;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: has;
}

<div
    className="c0"
  />,
]
`;

exports[`initialLetters renders invalid theme path 1`] = `
Array [
  .c0 {
  initial-letters: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`initialLetters renders negative number 1`] = `
Array [
  .c0 {
  initial-letters: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`initialLetters renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    initial-letters: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    initial-letters: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    initial-letters: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    initial-letters: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    initial-letters: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    initial-letters: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    initial-letters: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    initial-letters: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    initial-letters: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    initial-letters: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    initial-letters: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    initial-letters: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    initial-letters: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    initial-letters: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    initial-letters: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    initial-letters: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`initialLetters renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    initial-letters: July;
  }
}

@media April {
  .c0 {
    initial-letters: August;
  }
}

@media January {
  .c0 {
    initial-letters: September;
  }
}

@media April {
  .c0 {
    initial-letters: November;
  }
}

@media June {
  .c0 {
    initial-letters: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    initial-letters: July;
  }
}

@media April {
  .c0 {
    initial-letters: August;
  }
}

@media January {
  .c0 {
    initial-letters: September;
  }
}

@media April {
  .c0 {
    initial-letters: November;
  }
}

@media June {
  .c0 {
    initial-letters: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    initial-letters: January;
  }
}

@media April {
  .c0 {
    initial-letters: December;
  }
}

@media January {
  .c0 {
    initial-letters: April;
  }
}

@media April {
  .c0 {
    initial-letters: May;
  }
}

@media June {
  .c0 {
    initial-letters: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    initial-letters: January;
  }
}

@media April {
  .c0 {
    initial-letters: December;
  }
}

@media January {
  .c0 {
    initial-letters: April;
  }
}

@media April {
  .c0 {
    initial-letters: May;
  }
}

@media June {
  .c0 {
    initial-letters: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`initialLetters renders postitive number 1`] = `
Array [
  .c0 {
  initial-letters: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`initialLetters renders string 1`] = `
Array [
  .c0 {
  initial-letters: July;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: July;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: June;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: June;
}

<div
    className="c0"
  />,
]
`;

exports[`initialLetters renders valid theme path 1`] = `
Array [
  .c0 {
  initial-letters: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  initial-letters: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
